core-bundle/src/EventListener/CsrfTokenCookieSubscriber.php

```diff
@@ -70,7 +70,8 @@ public function onKernelResponse(ResponseEvent $event): void
 
         if ($this->requiresCsrf($request, $response)) {
             $this->setCookies($request, $response);
-        } else {
+        } elseif ($response->isSuccessful()) {
+            // Only delete CSRF token cookie, if response is successful (#2252)
```

Comment:
```suggestion
            // Only delete the CSRF token cookie if the response is successful (#2252)
```

There is no comma before "if" if the dependent clause follows the independent one. 🤓